src/wallet/api/wallet.cpp

```diff
@@ -605,18 +603,18 @@ bool WalletImpl::setPassword(const std::string &password)
     return m_status == Status_Ok;
 }
 
-std::string WalletImpl::address() const
+std::string WalletImpl::address(uint32_t account_index, uint32_t address_index) const
```

Comment:
I realized that I can make cryptonote_basic/subaddress_index.h independent of boost headers by moving the serialization code to cryptonote_basic/cryptonote_boost_serialization.h, so the wallet API now uses the struct `cryptonote::subaddress_index`. See commit fcc2b98